### DISCUSSION TITLE
An opportunity to build only changed files

### DISCUSSION THREAD

**Discussion by ekbfh** [mkdocs/mkdocs#2641] [Q&A]
Hi!

Is there any hack or trick to do so? My docs builds about half an hour and i want to decrease that time by having an artifacts from previous build on CI.
Is it possible?

**Comment by oprypin** [comment]
This is --dirty-reload basically - check that out. 

Caching is not implemented in any way.

Some thinking was done in https://github.com/mkdocs/mkdocs/issues/2384 but no results

**Reply by ekbfh** [reply]
Thx for the fast answer!

But no, i don't need a `serving` site, only `mkdocs build`
